packages/internal/rollup-plugin-bundle-manifest/src/index.ts

```diff
@@ -14,11 +14,11 @@ type ChunkItem = {
   type: 'chunk'
   mime: string
   sha256: string
-  dynamicImports: string[]
-  isDynamicEntry: boolean
-  isEntry: boolean
-  isImplicitEntry: boolean
-  name: string
+  // dynamicImports: string[]
+  // isDynamicEntry: boolean
+  // isEntry: boolean
+  // isImplicitEntry: boolean
+  // name: string
```

Comment:
Should we remove these?

Reply:
I left them there as an indication to future maintainers of what is available (but not exposed because not used). We can surely get rid of this.